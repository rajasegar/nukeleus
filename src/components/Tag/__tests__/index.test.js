import React from 'react';
import { renderWithTheme } from '../../../test-utils';

import { Tag } from '../index';

describe('Tag component test suite', () => {
  it('Should render Tag', () => {
    const container = renderWithTheme(<Tag variant="blue">Tag</Tag>);
    expect(container.firstChild).toMatchSnapshot();
  });
});
