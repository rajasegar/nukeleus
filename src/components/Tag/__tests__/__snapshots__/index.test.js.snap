// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tag component test suite Should render Tag 1`] = `undefined`;
